### DISCUSSION TITLE
How do you change the rotate in place / turn in place rate

### DISCUSSION THREAD

**Discussion by MagentaPuppet** [Sixze/ALS-Refactored#301] [Q&A]
I need to be able to dynamically change how fast the character rotates to the input direction when in velocity direction mode. I've searched through the als anim instance settings and als character settings files in blueprints but can't seem to find it.

**Comment by ameaninglessname** [comment]
I found this in code `UAlsAnimationInstanceSettings::TurnInPlace::StandingTurn90Left::PlayRate`, so there should be options in the `AIS_Als_Default` asset. (I can't open the editor at the moment)

**Reply by MagentaPuppet** [reply]
Already tried changing those value but they don't seem to do anything
